bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/common/PckGenerator.java

```diff
@@ -777,4 +797,28 @@ private static int timeToRampValue(int timeMSec) {
         }
         return ret;
     }
+
+    /**
+     * Converts duration in milliseconds to lcntimer value
+     * Source: https://www.symcon.de/forum/threads/38603-LCN-Relais-Kurzzeit-Timer-umrechnen
+     *
+     * @param ms time in milliseconds
+     * @return lcn timer value
+     */
+    private static Integer convertMsecToLCNTimer(double ms) {
```

Comment:
Primitive types should be used where possible.
```suggestion
    private static int convertMsecToLCNTimer(double ms) {
```